app/schemas.py

```diff
@@ -24,4 +24,57 @@ class UserlistIn(BaseModel):
     title: str
     ban: bool
     ignore: bool
-    user_ids: Union[None, list[int]]
\ No newline at end of file
+    user_ids: Union[None, list[int]]
+        
+        
+class Channel(BaseModel):
+    id: int
+    author_id: int
+    title: str
+    description: str
+    members_count: int
+    photo: Photo
+    create_time: str
+        
+        
+class Post(BaseModel):
+    post_id: str
+    author_id: int
+    channel_id: int
+    text: str
+    attachments: array
+    upload_time: str
+        
+        
+class Channel_Role(BaseModel):
```

Comment:
Наверное стоит назвать ChannelRole, везде называлось без "_" а тут вдруг так, да и впринципе так классы не называют насколько знаю
```suggestion
class ChannelRole(BaseModel):
```